examples/examples/cors_server.rs

```diff
@@ -68,10 +73,34 @@ async fn main() -> anyhow::Result<()> {
 }
 
 async fn run_server() -> anyhow::Result<(SocketAddr, HttpServerHandle)> {
-	let acl = AccessControlBuilder::new().allow_all_headers().allow_all_origins().allow_all_hosts().build();
+	// RPC access control that allows all hosts and all origins.
+	// Note: the access control does not modify the response headers,
+	// it only acts as a filter.
+	// If you need the ORIGIN header to be mirrored back in the response,
```

Comment:
this is a bit awkward i.e, if I enable origin filtering but forget to enable it in the `CORS middleware` the correct value is not sent back if one performs a preflight request....

but nothing really we can do about it... as we don't control the CORS middleware

Reply:
I think that's OK isn't it? I mean, if we assume that enabling origin filtering in jsonrpsee has nothing to do with CORS any more and will just block requests from origins people don't want accessing it.

(which I know sortof overlaps the CORS functionality, but I guess we're only keeping it here because we want to maintain consistency with WS connections and they don't have any CORS stuff)

Reply:
yupp, you are correct those are redundant.